.github/workflows/build_go_library.yaml

```diff
@@ -67,8 +67,26 @@ jobs:
           aws configure set aws_secret_access_key ${{ secrets.PROD_ASSET_SECRET_KEY }}
           aws configure set region ap-south-1
 
+      - name: Setup Go environment
+        uses: actions/setup-go@v4
+        with: 
+            go-version: 1.21.5
```

Comment:
What all repositories use `build_go_library.yaml` ? Did you test if this change is not causing any regression on them? Please share the workflows